### PR TITLE
Ddpb 2109 last saved by

### PR DESCRIPTION
**Pull request by slizzio**

### CODE REVIEW

File: src/AppBundle/Controller/Admin/Client/ReportController.php [hunk 1]
```diff
@@ -131,9 +131,9 @@ public function checklistAction(Request $request, $id)
     {
         $report = $this->getReport($id, ['report', 'report-checklist', 'checklist-information', 'last-modified', 'user']);
 
-        if (!$report->getSubmitted()) {
-            throw new DisplayableException('Cannot manage active report');
-        }
+//        if (!$report->getSubmitted()) {
```
Comment (elvisciotti) [inline src/AppBundle/Controller/Admin/Client/ReportController.php]: is this meant to be committed ?

File: tests/behat/features/deputy/03-report/01-102/24-lodging-checklist.feature [hunk 2]
```diff
@@ -70,7 +72,7 @@ Feature: Admin report checklist
     And the URL should match "/admin/report/\d+/checklist"
 
   @deputy @shaun
```
Comment (elvisciotti) [inline tests/behat/features/deputy/03-report/01-102/24-lodging-checklist.feature]: tag to remove

File: src/AppBundle/Controller/Admin/Client/ReportController.php (hunk 1 above)
Comment (slizzio) [reply src/AppBundle/Controller/Admin/Client/ReportController.php]: Nope, will remove.

Comment (slizzio) [reply src/AppBundle/Controller/Admin/Client/ReportController.php]: Put back in...

File: tests/behat/features/deputy/03-report/01-102/24-lodging-checklist.feature (hunk 2 above)
Comment (slizzio) [reply tests/behat/features/deputy/03-report/01-102/24-lodging-checklist.feature]: to remove. good spot

**Pull request merged**